app/templates/suppliers/duns_number.html

```diff
@@ -24,54 +24,55 @@
 {% endblock %}
 
 {% block main_content %}
-{% if form.duns_number.errors[0] == 'DUNS number already used' %}
-<div class="validation-masthead" role="group" aria-labelledby="validation-masthead-heading">
-  <h1 class="validation-masthead-heading" id="validation-masthead-heading">
-      A supplier account already exists with that DUNS number
-  </h1>
-  <p class="validation-masthead-description">
-      If you no longer have your account details, or if you think this may be an error, email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk?subject=DUNS%20number%20question" title="Please contact enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a>
-  </p>
-</div>
-{% endif %}
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    {% with heading = "Enter your DUNS number",
-            smaller = True %}
-      {% include "toolkit/page-heading.html" %}
-    {% endwith %}
+<div class="single-question-page">
+  <div class="grid-row">
+    <div class="column-two-thirds">
   
-    
-    <form method="POST" action="{{ url_for('.submit_duns_number') }}">
-      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-      <div class="dmspeak">
-        <p>If you registered your business with Companies House, you will automatically have been given a unique
-          DUNS number.</p>
-        <p>The Digital Marketplace uses this to check if your business already has a supplier account.</p>
-      </div>
-        <p class="lead">You can either:</p>
-        <ul class="list-bullet">
-          <li><a href="https://www.dnb.co.uk/duns-number/lookup.html" rel="external">find your DUNS number</a> on the
-            Dun &amp; Bradstreet website</li>
-          <li><a href="https://www.dnb.co.uk/duns-number/lookup/request-a-duns-number.html" rel="external">apply for
-            a DUNS number</a> if you don&rsquo;t have one</li>
-        </ul>
+      {% if form.duns_number.errors[0] == 'DUNS number already used' %}
+        {% with lede = "A supplier account already exists with that DUNS number",
```

Comment:
what's lede?

Reply:
It means "the opening sentence or paragraph of a news article, summarizing the most important aspects of the story."

It's a parameter that the `validation.html` macro looks for.

Reply:
I mean, I see it in validation form macro, but what does the name refer to, do you maybe know?

Reply:
Ahh, ok, thanks a lot :3